nose/utils_test.py

```diff
@@ -123,10 +123,12 @@ def test_openmp_summations():
     start = time.time()
     sum_a = np.dot(a,(b>1.0))
     np_timing = time.time()-start
+    print(sum_a)
```

Comment:
Could you revert these two print statements unless there is a reason for them to be present?

Reply:
Yes I can revert them. The reason why they are present is that on my local machine that test fails without the print statements (I never really understood why..). So I always have to manually add those print statements to run the nosetests. But since this problem is unrelated to this specific pull request, I can delete the print statements.